src/main/kotlin/io/github/rybalkinsd/kohttp/ext/HttpContextExt.kt

```diff
@@ -8,12 +8,15 @@ import java.net.URL
  * @author sergey
  */
 fun HttpContext.url(url: URL) {
-    if (url.protocol != "http" && url.protocol != "https") throw IllegalArgumentException("unexpected scheme: $scheme")
     scheme = url.protocol
+    if (scheme != "http" && scheme != "https")
```

Comment:
what is the reason to change line breaks in this file?

Reply:
Code style changes, it was too wide and uncomfortable to read